welcome-message/index.js

```diff
@@ -25,19 +106,35 @@ function init () {
 		fixedEl.hidden = true;
 		staticEl.hidden = true;
 	}
+}
 
+// this is the 'old' welcome functionality, where you only saw the sticky welcome once
+// we might go back to this once users are confident with new site
+function initOneTimeSticky () {
+	hideIfSegmentId();
 	if (Boolean(superstore.local.get(STORAGE_KEY)) === false && hasLocalStorage()) {
-		const closeButton = $('button', fixedEl);
-
+		const closeButton = fixedEl.querySelector('button');
```

Comment:
I'm not sure if we're restricted by o-expander in this but could we use a class selector here instead of a tag one? Maybe make a new one called `js-close-button` or something so it's decoupled from layout and markup implementation?